test/mongodb.test.js

```diff
@@ -64,6 +64,19 @@ describe('lazyConnect', function() {
     });
   });
 
+  it('should report url parsing error', function(done) {
+    const ds = global.getDataSource({
+      url: 'mongodb://xyz:@127.0.0.1:4/xyz_dev_db',
+    });
+
+    ds.on('error', function(err) {
+      should.exist(err);
+      err.name.should.equal('MongoNetworkError');
+      err.message.should.match(/failed to connect to server/);
```

Comment:
would [this code](https://github.com/strongloop/loopback-connector-mongodb/pull/531/files#diff-567d120f151f51afa9605ebcf0f20106R329) include the url in the error message?

Reply:
The url is included in the error by MongoDB driver.